examples/ffi/reqwest_exporter.cpp

```diff
@@ -0,0 +1,192 @@
+#include <cstdint>
+#include <cstdio>
+#include <cstdlib>
+#include <cstring>
+#include <datadog/common.h>
+#include <datadog/profiling.h>
+#include <memory>
+#include <thread>
+
+static ddog_CharSlice to_slice_c_char(const char *s) { return {.ptr = s, .len = strlen(s)}; }
```

Comment:
Does the cxx-ified version not end up with the `DDOG_CHARSLICE_C` and `DDOG_CHARSLICE_C_BARE` macros? Shouldn't need this unless you need an actual function for something.